app/views/patients/index.csv.erb

```diff
@@ -0,0 +1,8 @@
+<%=
```

Comment:
I think rendering as CSV is generally handled by the `respond_to`, and we just want to have this look like a regular-rails template -- no need for the CSV.generate in here. Instead, we can basically pretend like we're writing a regular HTML form by ditching the CSV generate and doing something like this:

```
<%= Patient::CSV_EXPORT_FIELDS.keys.join(',') %> # will render as `BSON ID, Identifier, Has alt contact?`
<% @patients.each do |patient| %>
  ...
<% end %>
```